### ISSUE TITLE
Cover Server

### ISSUE THREAD

**Issue by mamins1376** [abarisain/dmix#774]
Hi, It was a little hard for me to extract cover images and host the manually; so i wrote a server app to host covers dynamicly. Hope you enjoy :)

[MPDroid Cover Server](https://github.com/mamins1376/MPDroid-CoverServer)

I think it is good to add a link to Cover Server app, inside the MPDroid > Settings.

**Comment by bartkl** [comment]
Nice initiative to lower the bar for many :+1: A quick view at your code raises the following question though: does your script support other audio files than MP3s?

**Comment by mamins1376** [comment]
It should, mutagen uses ID3 metadata so every file should work, including audio files.

**Comment by bartkl** [comment]
I think you are mistaken. Mutagen offers an API for ID3 metadata handling, but for other standards also. FLAC files for instance usually use Vorbis tags. Or I am harshly mistaken somehow. Have you tested with FLAC files or some other non-MP3?

**Comment by mamins1376** [comment]
No. I would be great to add support for other audio files if mpdcs currently doesn't support them.

**Comment by bartkl** [comment]
Yeah, I like your initiatieve :). Currently I have little time but I might want to look into it one day.
